content/node-placement/docs/eks-node-placement.md

```diff
@@ -1,7 +1,7 @@
 # **EKS Node Placement**
 ## **Single AZ placement**
 
-AWS EKS clusters can span multiple AZs in a VPC. A Spark application whose driver and executor pods are distributed across multiple AZs can incur inter-AZ data transfer costs. To minimize or eliminate inter-AZ data transfer costs, you can configure the application to only run on the nodes within a single AZ.  In this example, we use the kubernetes [node selector](https://spark.apache.org/docs/latest/running-on-kubernetes.html#pod-spec) to specify which AZ should the job run on.
+AWS EKS clusters can span multiple AZs in a VPC. A Spark application whose driver and executor pods are distributed across multiple AZs can incur inter-AZ data transfer costs. To minimize or eliminate inter-AZ data transfer costs, you can configure the application to only run on the nodes within a single AZ. In this example, we use the kubernetes [node selector](https://spark.apache.org/docs/latest/running-on-kubernetes.html#pod-spec) and the `emr-job-submitter` classification to set a specific AZ for the driver, executor and job-runner pods.
```

Comment:
I think you should create separate paragraph with examples so that it stands out on its own.  Customers typically use node selectors for placing pods and we don't want to confuse between the two bcoz node selectors give more flexibility that may not be provided by emr classification. Create new paragraph with examples, provide details of what emr classification does and then mention what properties are allowed and if any are not allowed.

Reply:
Addressed comments in latest revision